stdlib/map.mli

```diff
@@ -158,6 +158,11 @@ module type S =
        where [k1 ... kN] are the keys of all bindings in [m]
        (in increasing order), and [d1 ... dN] are the associated data. *)
 
+    val fold_descending: (key -> 'a -> 'b -> 'b) -> 'a t -> 'b -> 'b
+    (** [fold_descending f m a] computes [(f k1 d1 (f k2 d2 ... (f kN dN a)...))],
+        where [k1 ... kN] are the keys of all bindings in [m]
+        (in increasing order), and [d1 ... dN] are the associated data. *)
```

Comment:
s/increasing/decreasing (below as well)

Reply:
I am open to suggestions for the name, of course.

Reply:
OK, it was pointed out to me that the order of `k1 d1 .. kN dN` is reversed, so the comment is correct, strictly speaking. However it may be better to leave the order of the indices unchanged and change "increasing" to "decreasing" instead.

Reply:
I can make this change but I founded more natural that the lowest element was numbered 1 and the highest was numbered N.